src/SFA.DAS.ReferenceData.PublicSectorOrgs.WebJob/Updater/IPublicSectorOrganisationHtmlScraper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HtmlAgilityPack;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.ReferenceData.Domain.Models;
+
+namespace SFA.DAS.ReferenceData.PublicSectorOrgs.WebJob.Updater
+{
+    public interface IPublicSectorOrganisationHtmlScraper
+    {
+        PublicSectorOrganisationLookUp Scrape(string url, ILog logger);
+    }
+
+    public class PublicSectorOrganisationHtmlScraper : IPublicSectorOrganisationHtmlScraper
+    {
+        public PublicSectorOrganisationLookUp Scrape(string url, ILog logger)
```

Comment:
It would be nice to have tests for this. I know you've just refactored it out but it's a chance to improve it. It could take an injectable thing that provides a stream and the thing scrapes the stream. Optional but sounds fun if you have time.